pkg/ovs/openflow/ofctrl_packetin_test.go

```diff
@@ -219,3 +227,33 @@ func TestParsePacketIn(t *testing.T) {
 		})
 	}
 }
+
+func TestGetTCPDataWithoutOptions(t *testing.T) {
+	type args struct {
+		tcp        protocol.TCP
+		expectData []byte
+	}
+	tests := []struct {
+		name string
+		args args
+	}{
+		{
+			name: "GetTCPDNSData",
+			args: args{
+				tcp: protocol.TCP{
+					HdrLen: 6,
+					Data:   []byte{1, 2, 3, 4, 0, 0, 5},
```

Comment:
the length field should describe the length to mock the dns data.

Reply:
Done.